packages/theme/src/lib/light-theme.ts

```diff
@@ -258,6 +258,14 @@ export const lightTheme = {
     },
   },
   components: {
+    FeedbackContainer: {
```

Comment:
Do we want to prefix these so all availity components are grouped? `AvFeedback` with a container slot

Reply:
Yeah, that sounds like a good idea.

Reply:
Done